scriptmodules/libretrocores/lr-mame2003.sh

```diff
@@ -59,17 +59,19 @@ function configure_lr-mame2003() {
         for mame_sub_dir in cfg ctrlr diff hi memcard nvram; do
             mkRomDir "$mame_dir/$dir_name/$mame_sub_dir"
         done
-
-        # lr-mame2003-plus also has an artwork folder
-        [[ "$md_id" == "lr-mame2003-plus" ]] && mkRomDir "$mame_dir/$dir_name/artwork"
     done
 
     mkUserDir "$biosdir/$dir_name"
     mkUserDir "$biosdir/$dir_name/samples"
-
-    # copy hiscore.dat
+    
+    # copy hiscore.dat and cheat.dat
     cp "$md_inst/metadata/"{hiscore.dat,cheat.dat} "$biosdir/$dir_name/"
     chown $user:$user "$biosdir/$dir_name/"{hiscore.dat,cheat.dat}
+    
+    # lr-mame2003-plus also has an artwork folder
+    [[ "$md_id" == "lr-mame2003-plus" ]] && mkRomDir "$biosdir/$dir_name/artwork"
+    [[ "$md_id" == "lr-mame2003-plus" ]] && cp "$md_inst/metadata/artwork/*.*" "$biosdir/$dir_name/artwork/"
```

Comment:
quoting *.* won't work.

Reply:
Thanks you -- I only deal with Bash occasionally. I've tried to follow your suggestion here and removed the `*.*` but I'm not positive I understand what you mean by quoting it.

Can I use a single `*` and still keep it in the quotes as in my most recent commit?

Reply:
@markwkidd , you need to leave the globbing (`*`) outside of the quotes, otherwise they will be interpreted as asterisks instead of expanding. In fact it is good practice to quote the minimum possible (around the variables). For example:
```
cp "$md_inst"/metadata/artwork/* "$biosdir/$dir_name/artwork/"
```

Reply:
Thanks @hhromic ! Now I understand what's up with the asterisk and I've pushed a commit to move it outside the double quotes

Reply:
happy to help!  and thanks you too for the great `lr-mame2003-plus` core, it's a delight hands down!